lib/internal/per_context/domexception.js

```diff
@@ -49,14 +49,23 @@ const disusedNamesSet = new SafeSet()
   .add('ValidationError');
 
 class DOMException {
-  constructor(message = '', name = 'Error') {
+  constructor(message = '', name = 'Error', cause = undefined) {
```

Comment:
Is this signature preferable over any of these?
```suggestion
  constructor(message = '', name = 'Error', options = { cause }) { // pseudocode
```
or
```suggestion
  constructor(message = '', nameOrOptions = 'Error') {
```